### ISSUE TITLE
Globe Developer Conference - Hot Starts - August 8, 2023

### ISSUE THREAD

**Issue by rh-crosero** [openshift-instruqt/instruqt#268]
**Name the event**
Globe Developer Conference

**Date and time of the event**
August 8, 2023 08:00 to 18:00 PHT (Philippine Time)

**Event duration**
10 Hours

**Content Request**

30 Hotstarts for each please: 
1. Getting Started with OpenShift for Developers - https://developers.redhat.com/courses/foundations/getting-started
2. Getting Started with OpenShift Pipelines - https://developers.redhat.com/courses/gitops/getting-started-openshift-pipelines
3. Deploying Applications From Source - https://developers.redhat.com/courses/openshift/deploying-python
4. Red Hat OpenShift 4.12 Playground - https://developers.redhat.com/courses/openshift-412-playground

**Comment by cedricclyburn** [comment]
@rh-crosero Good to go, best of luck!

<img width="721" alt="image" src="https://github.com/openshift-instruqt/instruqt/assets/54345140/b0658502-20f8-4b90-ae12-e02312271965">

**Comment by cedricclyburn** [comment]
Whoops, just edited the date, I had miscalculated time zone:
<img width="713" alt="image" src="https://github.com/openshift-instruqt/instruqt/assets/54345140/9e4aa19a-8829-40f4-af49-f9d9cca1ad0c">

**Comment by rh-crosero** [comment]
Hi Cedric, 

Sorry about this last-minute ask; we have to change the hot start requests to the following, as we realized some originally selected could take hours, and we might not be able to keep some people in the event for that long; please see the links we'll need the hot starts on, still for 30 each. Thank you, and appreciate your quick revert on this!

1. https://developers.redhat.com/courses/openshift/getting-started
2. https://developers.redhat.com/courses/gitops/getting-started-openshift-pipelines
3. https://developers.redhat.com/courses/openshift/deploying-python
4. https://developers.redhat.com/courses/get-started-argo-cd-and-gitops-red-hat-openshift

:

**Comment by evanshortiss** [comment]
@rh-crosero, I've updated the Hot Starts per the attached image. The hot starts will begin to provision at 0700 PHT After this they will be available for 11 hours from that time until 1800 PHT.

EDIT: I've corrected the Hot Starts to use the correct pipelines track per Cedric's original list. The names in Instruqt don't line up exactly with what you see externally :) 

![Screenshot 2023-08-07 at 10 20 18 AM](https://github.com/openshift-instruqt/instruqt/assets/1303687/5a787648-a921-4b84-9328-97aad7d5c160)
